docs/about.md

```diff
@@ -21,13 +21,17 @@ This guide was written by staff members of the [Nintendo Homebrew Discord Server
     - **Atmosphere-NX** for [Atmosphere](https://github.com/Atmosphere-NX/Atmosphere).
     - **switchbrew** for [nx-hbloader](https://github.com/switchbrew/nx-hbloader) and [nx-hbmenu](https://github.com/switchbrew/nx-hbmenu).
     - **nwert** and **CTCaer** for [Hekate](https://github.com/CTCaer/hekate).
+    - **WerWolf** for [EdiZon](https://github.com/WerWolv/EdiZon/releases).
```

Comment:
``WewWolf`` -> ``WerWolv``

Reply:
The author has their display name on everything as WerWolv, why the change?

Reply:
It's called a typo.

Force pushed and fixed.